powerauth-nextstep/src/main/java/io/getlime/security/powerauth/app/nextstep/service/CredentialService.java

```diff
@@ -625,6 +626,7 @@ public CredentialSecretDetail createCredential(UserIdentityEntity user, Credenti
         credential.setEncryptionAlgorithm(protectedCredentialValue.getEncryptionAlgorithm());
         credential.setHashingConfig(credentialDefinition.getHashingConfig());
         credential.setTimestampLastCredentialChange(new Date());
+        credential.setTimestampLastUsernameChange(new Date());
```

Comment:
Store the `new Date()` in variable to avoid having different values for the two timestamps.